<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>400</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>159</red>
        <green>159</green>
        <blue>159</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>159</red>
        <green>159</green>
        <blue>159</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>159</red>
        <green>159</green>
        <blue>159</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>159</red>
        <green>159</green>
        <blue>159</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Evan's Awesome TrinityCore GUI</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="AuthStartButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="authServerText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>221</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="worldServerText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="AuthLabel">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>11</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>authserver</string>
    </property>
   </widget>
   <widget class="QLabel" name="WorldLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>worldserver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AuthStopButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WorldStartButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WorldStopButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>96</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="worldSendButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="worldSendText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <action name="actionList">
   <property name="text">
    <string>list</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>AuthStartButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>authStartButtonClickHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AuthStopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>authStopButtonClickHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WorldStopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>worldStopButtonClickHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WorldStartButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>worldStartButtonClickHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>worldSendButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>worldSendButtonClickHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>worldSendText</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>worldSendButtonClickHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>isVisible()</signal>
  <slot>authStartButtonClickHandler()</slot>
  <slot>authStopButtonClickHandler()</slot>
  <slot>worldStopButtonClickHandler()</slot>
  <slot>worldStartButtonClickHandler()</slot>
  <slot>authSendButtonClickHandler()</slot>
  <slot>worldSendButtonClickHandler()</slot>
  <slot>reloadButtonClickHandler()</slot>
 </slots>
</ui>
